sdk/servicebus/service-bus/src/core/managementClient.ts

```diff
@@ -831,7 +836,8 @@ export class ManagementClient extends LinkEntity<RequestResponseLink> {
   async updateDispositionStatus(
     lockToken: string,
     dispositionType: DispositionType,
-    options?: DispositionStatusOptions & SendManagementRequestOptions
+    // TODO: mgmt link retry<> will come in the next PR.
```

Comment:
Is it __really__?

Reply:
Well...I should probably file an issue for that.

Reply:
Issue added here: https://github.com/Azure/azure-sdk-for-js/issues/14885